File: src/kyu_8/string_templates_bug_fixing_number_5/java/BuildStringTest.java
package kyu_8.string_templates_bug_fixing_number_5.java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class BuildStringTest {
    @Test
    public void fixedTests() {
        assertEquals(Kata.buildString("Cheese", "Milk", "Chocolate"), "I like Cheese, Milk, Chocolate!");
        assertEquals(Kata.buildString("Cheese", "Milk"), "I like Cheese, Milk!");
        assertEquals(Kata.buildString("Chocolate"), "I like Chocolate!");
    }
}

File: src/kyu_8/string_templates_bug_fixing_number_5/js/buildStringTest.js
const buildString = require("./buildString");
const assert = require('chai').assert3describe('Fixed Tests', function () {
    it('Fixed Tests', function () {
        assert.strictEqual(buildString('Cheese', 'Milk', 'Chocolate'), 'I like Cheese, Milk, Chocolate!', 'Return the correct String');
        assert.strictEqual(buildString('Cheese', 'Milk'), 'I like Cheese, Milk!', 'Return the correct String');
        assert.strictEqual(buildString('Chocolate'), 'I like Chocolate!', 'Return the correct String');
    })
});